src/api/models.py

```diff
@@ -183,30 +183,29 @@ class ShotModel(SQLModel, table=True):
     )
 
     reference_shot: Optional[int] = Field(
-        nullable=True,
         description='Reference shot ID used as the basis for setting up this shot, if used. e.g. "30420"',
     )
 
     scenario: Optional[int] = Field(
-        nullable=True, description="The scenario used for this shot."
```

Comment:
Why do we need to add `nullable=True` to all these fields? Was this not working before?